src/BotManager.php

```diff
@@ -502,7 +502,8 @@ protected function defaultGetUpdatesCallback($get_updates_response): string
                 $text    .= ";{$update_content->getQuery()}";
             } elseif ($update_content instanceof CallbackQuery) {
                 /** @var CallbackQuery $update_content */
-                $chat_id = $update_content->getMessage()->getChat()->getId();
+                $message = $update_content->getMessage();
+                $chat_id = ($message!==null && $message->getChat()!==null) ? $message->getChat()->getId() : null;
```

Comment:
To fix the code sniffer errors in the build, use something like this:
```php
$message = $update_content->getMessage();
if ($message && $message->getChat()) {
    $chat_id = $message->getChat()->getId();
}
```

Reply:
Please add note about `composer check-code` requirement to `pull request template` so i don't remember to properly format according to project rules next time.

To be exact, we need with help of `instanceof` check interface of `$message` and interface of `$message->getChat()` before use them.

Reply:
> Please add note about composer check-code requirement to pull request template

Absolutely, thanks for pointing that out 🙏 

Checking the type isn't necessary, as the methods themselves ensure that the right type is returned. You also don't need to check for `null`, because it's either `null` or the correct object, so the `if` statement works perfectly without that specific check.